### PR TITLE
Android CI update for tests

### PR DESCRIPTION
**Pull request by Akuien**
I want to test if we can use the Android Ci to integrate the tests and if possible generate tests reports.

### :ballot_box_with_check: Definition of Done checklist
- [x] Meaningful title and valuable description in pull request
- [x] Code change tested manually if not covered by automated tests
- [x] Pull request linked to one or more issues/stories
- [x] Acceptance criteria of the linked stories satisfied
- [x] Code change comprehended by those who approved it
- [x] State of target branch maintained or improved after the code change

### CODE REVIEW

File: .github/workflows/android.yml [hunk 1]
```diff
@@ -31,3 +31,46 @@ jobs:
...
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3.1.0
+        with:
+          distribution: 'temurin'
+          java-version: '11'
+      - name: Run instrumented tests
+        uses: reactivecircus/android-emulator-runner@v2.24.0
+        with:
+          api-level: 29
+          script: cd CarEshipApp && ./gradlew connectedAndroidTest
```
Comment (d0nate110) [inline .github/workflows/android.yml:55]: Try with ./gradlew connectedCheck instead

**Review by d0nate110 — CHANGES_REQUESTED** [review]
add - uses: actions/checkout@v2

File: .github/workflows/android.yml (hunk 1 above)
Comment (Akuien) [reply .github/workflows/android.yml:58]: Oh yeah, am gonna that. Thanks.

**Comment by d0nate110** [comment]
Now seems like there is a problem with Task :app:processDebugGoogleServices which failed

**Comment by d0nate110** [comment]
File google-services.json is missing

**Comment by Akuien** [comment]
Yeah, it says google-services.json file is missing.

**Comment by Akuien** [comment]
> File google-services.json is missing

I might have an idea, Hahah.

**Comment by d0nate110** [comment]
Nice fix!!

**Review by d0nate110 — APPROVED** [review]

**Pull request merged**